chainer/links/connection/scale.py

```diff
@@ -0,0 +1,83 @@
+import chainer
+from chainer.functions.math import scale
+from chainer import link
+from chainer.links.connection import bias
+
+
+class Scale(link.Chain):
+    """Broadcasted elementwise product with learnable parameter.
+
+    Computes a elementwise product as :func:`~chainer.functions.scale`
+    function does except that its second input is a learnable weight parameter
+    :math:`W` the link has.
+
+    Args:
+        axis (int): The first axis of the first input of
+            :func:`~chainer.functions.scale` function along which its second
+            input is applied.
+        W_shape (tuple of ints): Shape of learnable weight parameter. If
+            ``None``, a weight parameter needs to be given explicitly to its
+            ``__call__`` method's second input.
+        bias_term (bool): Whether to also learn a bias (equivalent to Scale
+            link + Bias link).
+        bias_shape (tuple of ints): Shape of learnable bias. If ``W_shape`` is
+            ``None``, this should be given to determine the shape. Otherwise,
+            the bias has the same shape ``W_shape`` with the weight parameter
+            and ``bias_shape`` is ignored.
+
+    .. seealso:: See :func:`chainer.functions.scale` for details.
+
+    Attributes:
+        W (~chainer.Variable): Weight parameter.
+        bias (~chainer.links.Bias): Bias term.
```

Comment:
ditto (`W` and `bias` parameters can not exist depending on the init parameters).